addons/l10n_es/__manifest__.py

```diff
@@ -47,9 +47,11 @@
         'data/mod390/mod390_section6.xml',
         'data/mod390/mod390_section7.xml',
         'data/mod390/mod390.xml',
+        'data/mod420.xml',
```

Comment:
So this PR is actively adding a new report ? 
If so, I feel like this should at the very least be a separate commit.